### PR TITLE
Feat/academy introduction

### PR DESCRIPTION
**Pull request by MdechampG**
This PR aims at adding an introduction to the Academy overview. It lists the material that it is advised to read before delving into the academy. 

Capture d’écran 2024-01-29 à 11 38 09

Capture d’écran 2024-01-29 à 11 38 26

Capture d’écran 2024-01-29 à 11 38 38

## Summary by CodeRabbit

- **Documentation**
	- Updated the Academy overview to include an image, a new section on the learning journey, and a rephrased introduction to the OKP4 protocol. Also, restructured the content into Fundamental Concepts and Technical Implementation for better clarity.

### CODE REVIEW

**Review by ccamel — CHANGES_REQUESTED** [review]
The content is excellent. Great! 

I've left a few suggestions to enhance the presentation slightly.

File: docs/academy/overview.mdx [hunk 1]
```diff
@@ -8,9 +8,38 @@ Welcome to the OKP4 Academy, your gateway to the cutting-edge world of off-chain
...
+
+OKP4 is a public blockchain built to share and orchestrate any off-chain resource. Its architecture is specifically designed to:
+
+- Share any off-chain resource (datasets, algorithms, services...).
+- Define and enforce highly customized rules on-chain (access rights, business models, governance models...).
+- Build applications leveraging the shared resources according to the on-chain rules.
+
+:::tip
+Before delving into the academy modules, we strongly recommend familiarizing yourself with essential sections of the OKP4 Whitepaper:
+1. The [Abstract](https://docs.okp4.network/whitepaper/abstract): Gain a broad overview of the OKP4 protocol to understand its fundamental principles.
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/overview.mdx]: You're right, I changed it.

File: docs/academy/overview.mdx [hunk 2]
```diff
@@ -8,9 +8,38 @@ Welcome to the OKP4 Academy, your gateway to the cutting-edge world of off-chain
...
+2. The [Introduction](https://docs.okp4.network/whitepaper/introduction): Explore the challenges within the knowledge economy that OKP4 addresses.  
+3. The [Solution](https://docs.okp4.network/whitepaper/solution): Discover the high-level concepts and key features underlying the OKP4 protocol. 
+:::
+
+The Academy is structured into two main parts to cater to diverse learning needs:
+
+**Part 1 - Fundamental Concepts:**
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/overview.mdx]: Done for the first part. I'll add the second part later (possible conflict with another PR).

File: docs/academy/overview.mdx [hunk 3]
```diff
@@ -8,9 +8,38 @@ Welcome to the OKP4 Academy, your gateway to the cutting-edge world of off-chain
 
 Throughout this concise journey, we will guide you through the core principles of the OKP4 protocol, providing you with a swift understanding of its functions. Prepare to immerse yourself in OKP4, where the focus lies on optimizing resource sharing and creating new knowledge. Are you ready to explore the forefront of off-chain resource management? Follow us through this Academy to unravel the intricacies of OKP4.
 
+![overview-1](/img/content/academy/overview-1.webp)
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/overview.mdx:11]: Indeed ! Done ✅

**Comment by MdechampG** [comment]
> @MdechampG The linter is complaining that the image file in use is actually a `JPEG`, despite the extension indicating it's a `WEBP`.
> 
> ```shell
> $ file static/img/content/academy/overview-1.webp
> static/img/content/academy/overview-1.webp: JPEG image data, Exif standard: [TIFF image data, big-endian, direntries=6, orientation=upper-left, xresolution=86, yresolution=94, resolutionunit=2], baseline, precision 8, 340x340, components 3
> ```

It should be good now 🤞🏻

**Review by ccamel — APPROVED** [review]
Great! :+1:

**Review by JulienMassonnat — COMMENTED** [review]
Great job !

**Pull request merged**